.github/ISSUE_TEMPLATE/als-----wil-ik----.md

```diff
@@ -8,24 +8,28 @@ about: Beschrijving van een feature/user story
 
 **Definition of ready**
 - [ ] Iedereen in het team begrijpt de user story
+- [ ] de gewenste (aanvulling op de) functionaliteit van de API's duidelijk en beschreven is.
 - [ ] Is klein genoeg (maximaal 1/5 van sprint)
-- [ ] Product Owner akkoord
+- [ ] Product Owner akkoord en voorzien van prioriteit (mag alleen afgevinkt worden door Eelco)
 - [ ] Voorzien van acceptatiecriteria (duidelijk en testbaar)
 - [ ] Voorzien van Definition of Done (duidelijk en testbaar)
 - [ ] Voorzien van taken
 - [ ] Idee hebben van hoe deze user story kan worden gedemonstreerd.
 - [ ] _**Userstory is ingevuld op template architectuurschets**_
-- [ ] **_Userstory is voorzien van veldmapping naar RGBZ2_**
-- [ ] **_Userstory past op wenselijk gebruik ZGW volgens GEMMA 2_**
+- [ ] **_Userstory is voorzien van veldmapping naar RGBZ2_** (mag alleen afgevinkt worden door Arjan)
+- [ ] **_Userstory past op wenselijk gebruik ZGW volgens GEMMA 2_** (mag alleen afgevinkt worden door Jeffrey)
 - [ ] Vastgelegd in Github en geplaatst in kolom ready
 
 **Definition of done**
 - [ ] er is een OAS 3.0 specificatie
-- [ ] de specificatie is gepubliceerd leesbaar
+- [ ] de specificatie is gepubliceerd leesbaar (functioneel en technisch) 
 - [ ] er is een referentieimplementatie
 - [ ] de DSO URI- en API-strategie worden gevolgd of afwijkingen zijn vastgelegd als ontwerp keuze
 - [ ] eventueel gemaakte ontwerp keuzes zijn gedocumenteerd
 - [ ] er zijn geen (nieuwe) conflicten met ontwerp keuzes in [BIP](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/blob/master/docs/design_decisions.md)
+- [ ] review heeft plaatsgevonden (dus reviewers hebben voldoende en vindbare info om te kunnen reviewen (dus duidelijke verwijzing in issue/user story naar deze documentatie)
+
+_Zorg ervoor dat in de documentatie duidelijk is welke wijzigingen zijn doorgevoerd t.a.v. specifieke user story, zodat het voor iedereen duidelijk is wat er gewijzigd of toegevoegd is._
```

Comment:
Ik bedoel 'm precies omgekeerd met deze zin: 'gedocumenteerd is welke wijzigingen zijn doorgevoerd in de API's a.g.v. de user-story'. Dus, maar van cq. in een userstory duidelijk wat er zoal gewijzigd is n.a.v. die userstory. Doe dat niet in de (functionele en technische) documentatie (m.u.v. de desbetreffende tabel in de functionele documentatie). Dit moet het eenvoudig te vinden maken wat er als gevolg van een userstory gewijzgd is en wat er dus gereviewd moet worden. Ik pleit er nog steeds voor hiervan een apart punt te maken (het geeft richtlijnen voor het bijschrijven de userstory) i.p.v. een opmerking.

Reply:
Prima